docs/guide/using_existing_contracts.rst

```diff
@@ -37,77 +37,79 @@ If you do not have ABI statically, but you know the interface of the contract on
 Fees
 ----
 
-starknet.py requires you to specify amount of Wei you
-are willing to pay either when making ``.invoke()`` transactions or when preparing
-function calls with ``.prepare()``.
+.. currentmodule:: starknet_py.contract
+
+Starknet.py requires you to specify amount of Wei (for V1 transaction) or Fri (for V3 transaction) you
+are willing to pay when executing either :meth:`~ContractFunction.invoke_v1` or :meth:`~ContractFunction.invoke_v3` transactions.
```

Comment:
How about we mention right away at the top that `"Alternatively, you can estimate fee automatically by..."`

Reply:
I'd prefer to keep the current version, as incorporating all the information about fee estimation here would overly extend the `Fees` section.

Reply:
Current wording with `requires` implies there's no way to send an invoke transaction without setting max fee. The intention is not to provide all the information, but merely show the user know this is possible.

Reply:
Does it make sense to add `Alternatively, you can estimate fee automatically, as described in the 'Automatic fee estimation' section below.`?

Reply:
Sounds good 👍🏻